spikeinterface/sorters/runsorter.py

```diff
@@ -272,14 +281,13 @@ def run_sorter_container(sorter_name, recording, mode, container_image, output_f
     container_client.start()
 
     # check if container contains spikeinterface already
-    # this do not work with singularity:
-    # cmd = 'python -c "import spikeinterface; print(spikeinterface.__version__)"'
-    # this approach is better
-    cmd = ['python', '-c', 'import spikeinterface; print(spikeinterface.__version__)']
-    res_output = container_client.run_command(cmd)
+    cmd_1 = ['python', '-c', 'import spikeinterface; print(spikeinterface.__version__)']
+    cmd_2 = ['python', '-c', 'from spikeinterface.sorters import run_sorter_local']
```

Comment:
Strange, why ?
I don't see how cmd1 would work and not cmd2.

Reply:
I added the cmd_2 to ensure that not only the minimal spikeinterface package is available, but also the `run_sorter_local`. This can be an issue if spikeinterface was for example only installed with the base requirements, then the `cmd_1` succeeds, but `cmd_2` fails due to missing requirements.

Reply:
ok.